README.md

```diff
@@ -132,7 +146,7 @@ Create a task JSON file (exemplary file in examples/tasks/users-file.json):
 
 Create a task:
 ```
-$ snapctl task create -t $SNAP_USERS_PLUGIN_DIR/examples/tasks/users-file.json
+$ snapctl task create -t users-file.json
```

Comment:
Above there was used `$SNAP_PATH/bin/snapctl`, now only `snapctl` -> consistency should be kept